newreply.php

```diff
@@ -277,7 +277,7 @@
 	}
 }
 
-$reply_errors = $quoted_ids = '';
+$reply_errors = $quoted_ids = array();
```

Comment:
`$reply_errors` should be kept as string - otherwise https://github.com/mybb/mybb/blob/mybb_1808/newreply.php#L882 incorrectly evaluates as true